perf/src/sigverify.rs

```diff
@@ -320,13 +334,14 @@ pub fn get_checked_scalar(scalar: &[u8; 32]) -> Result<[u8; 32], PacketError> {
 }
 
 pub fn ed25519_verify(
-    batches: &[Packets],
+    batches: &mut [Packets],
```

Comment:
You will need to mark gpu-verified packets that fail verification as discarded too

Reply:
those will still get marked as discarded by the old path in `mark_disabled` 

```
let r = verify_shreds_gpu(&batches, &leader_slots, &self.recycler_cache);
sigverify::mark_disabled(&mut batches, &r);
```

Reply:
What's the point of marking as disabled within `verify_shreds` if we are still marking as discarded outside?